nailgun/entity_fields.py

```diff
@@ -265,4 +266,4 @@ class URLField(StringField):
 
     def gen_value(self):
         """Return a value suitable for a :class:`URLField`."""
-        return gen_url()
+        return gen_url(subdomain=gen_alpha())
```

Comment:
9gag and 4chan fans might get upset about this solution..how about this?
```python
return gen_url(subdomain=gen_alphanumeric())
```

also, how about including a `.` to support multiple-domain format (like `foo.bar.org`)?
of course it must not be at the beginning or at the end of the string